codespell_lib/data/dictionary.txt

```diff
@@ -18555,10 +18561,12 @@ provate->private, provide,
 provded->provided
 provder->provider
 provdided->provided
+provdidet->provided, provider, provident,
 provdie->provide
 provdied->provided
 provdies->provides
 provding->providing
+provedd->proved
```

Comment:
I wonder if this should suggest `provided`?

Reply:
aa79ac5